lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -152,6 +152,23 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       const estValuesEl = this.dom.createChildOf(metricAuditsEl, 'div', 'lh-metrics__disclaimer');
       const disclaimerEl = this.dom.convertMarkdownLinkSnippets(strings.varianceDisclaimer);
       estValuesEl.appendChild(disclaimerEl);
+
+      // Add link to score calculator
+      const calculatorLink = this.dom.createChildOf(estValuesEl, 'a');
+      calculatorLink.textContent = strings.calculatorLink;
+      const v5andv6metrics = /** @type {LH.ReportResult.AuditRef[]} */ ([
```

Comment:
would be good to not override tsc here and deal with the possible undefineds. The benefit is it'll be backward and forward (when we drop these metrics) compatible :)

Reply:
k done